tests/server/test_models.py

```diff
@@ -0,0 +1,296 @@
+"""Test cases for server models."""
+
+import pytest
+from sqlmodel import Field, Session, SQLModel, create_engine
+from sqlmodel.pool import StaticPool
+
+from lilypad.server.models import (
+    ActiveVersionPublic,
+    BaseSQLModel,
+    FunctionCreate,
+    FunctionPublic,
+    FunctionTable,
+    ProjectCreate,
+    ProjectPublic,
+    ProjectTable,
+    PromptCreate,
+    PromptPublic,
+    PromptTable,
+    Provider,
+    ResponseFormat,
+    Scope,
+    SpanCreate,
+    SpanPublic,
+    SpanTable,
+    VersionCreate,
+    VersionPublic,
+    VersionTable,
+)
+
+
+@pytest.fixture
+def fixture_engine():
+    """Create an in-memory database engine for testing."""
+    engine = create_engine(
+        "sqlite://", connect_args={"check_same_thread": False}, poolclass=StaticPool
+    )
+    SQLModel.metadata.create_all(engine)
+    return engine
+
+
+@pytest.fixture
+def session(fixture_engine):
+    """Create a new database session for testing."""
+    with Session(fixture_engine) as session:
+        yield session
+        session.rollback()
+
+
+def test_base_sql_model():
```

Comment:
these are all missing return annotations of `None` please update all tests to return `None`